### ISSUE TITLE
Not very pleasing rendering

### ISSUE THREAD

**Issue by shaomeng** [NCAR/VAPOR#1737]
To Reproduce: 
1. Open VAPOR, load data `/glade/p/cisl/vast/vapor/Bugs/cm1.nc`
2. Create a volume renderer, adjust the left most opacity control point to the bottom.
3. Enable the DVR, the rendering looks plausible. 
4. Create an IsoSurface renderer. Enable one iso value. Adjust that iso-value to 2.
3. The resulting rendering is like below. It doesn't look very pleasing...

![no-pleasing](https://user-images.githubusercontent.com/814885/60052490-a5a5aa00-9692-11e9-8705-4009b2200abd.png)

**Comment by shaomeng** [comment]
Well, I'd say there's an improvement, but the very noticeable noise dots are not ideal.
(you'll need to click on the screenshot below.)

![ddd](https://user-images.githubusercontent.com/814885/60624210-ecd22000-9da1-11e9-818f-f97d787e2bb0.png)

**Comment by StasJ** [comment]
@shaomeng @clyne I mentioned at our last meeting that it appears that this is the only dataset that is still giving us artifact problems. While testing OSPRay I opened this dataset in the OSPRay regular grid renderer and noticed that the same "bad pixels" were appearing in the same area. This lead me to question whether these were "bad pixels" or rather artifacts of the dataset. I did some further testing and it appears that this is in fact noise in the uninitialized portion of the dataset.
